### PR TITLE
Terraform landing page and tutorial

### PR DESCRIPTION
**Pull request by cloudjumpercat**
Adding Terraform landing page

Need to figure out how to add tables (or a good way to display the info https://github.com/Kong/terraform-provider-konnect?tab=readme-ov-file#capabilities)

Preview link: https://deploy-preview-36--kongdeveloper.netlify.app/terraform/

### CODE REVIEW

File: app/_includes/landing_pages/structured_text.md [hunk 1]
```diff
@@ -24,4 +24,12 @@
 </ul>
 {% endif %}
 
+{% if item.type == "toggle" %}
+
+<div>
+<summary>{{ item.title | markdown }}</summary>
+<p>{{ item.text | markdown }}</p>
+</div>
+{% endif %}
```
Comment (cloudjumpercat) [inline app/_includes/landing_pages/structured_text.md:33]: @fabianrbz I tried to add those toggles (similar to the one on this page https://docs.konghq.com/konnect/getting-started/add-api/#main), but they aren't formatted correctly.

Comment (fabianrbz) [reply app/_includes/landing_pages/structured_text.md:33]: @cloudjumpercat you need to wrap that in a `<details>` tag

**Comment by lena-larionova** [comment]
@cloudjumpercat can you rebase this?

**Comment by cloudjumpercat** [comment]
After going through the content in this PR, I'm going to close it for now. The how to content was basically copy/pasted from the Terraform GitHub repo readme and I hadn't revised it yet. The landing page content was sparse and I don't think the diagram I made was useful, so I think I'll just close this.

**Pull request closed without merging**